testFixture/test.yaml

```diff
@@ -20,9 +20,9 @@ spec:
   build:
     source:
       ddl: >
-        CREATE DATABASE {name} OPTIONS (ANNOTATION 'provide your description
+        CREATE DATABASE example OPTIONS (ANNOTATION 'provide your description
```

Comment:
Quite simple, leave the code in better quality than you found it. {name} was a placeholder in the past and in a real world that placeholder would have been replaced by the name of the VDB. It is not linked to the original issue.

Reply:
great.

would have been best to provide it in a differrent PR or at least a different commit as it is not related so that when reviewing or looking in the history we won't wonder why it is in the same commit than removing spring support.